src/test/javascript/spec/component/course/course-group.component.spec.ts

```diff
@@ -115,6 +115,7 @@ describe('Course Management Detail Component', () => {
                 expect(users).toEqual([courseGroupUser]);
             });
             expect(searchStub).toHaveBeenCalledWith(loginOrName);
+            expect(searchStub).toHaveBeenCalledTimes(1);
```

Comment:
```suggestion
            expect(searchStub).toHaveBeenCalledTimes(1);
            expect(searchStub).toHaveBeenCalledWith(loginOrName);
```
When testing we should always go from abstract to specific.
=> If the number of calls is wrong we don't need to test the calls themselves to see if the test should fail
=> If we would first check the calls themselves, there could be one of them correct. So we'd have to check the number of calls afterwards.

Reply:
Please also apply for other occurrences